modules/client/src/connext.ts

```diff
@@ -626,7 +622,7 @@ export class ConnextInternal extends ConnextChannel {
     // TODO: poller should not be completely blocking, but safe to leave for now
     // because the channel should be blocked
     try {
-      await new Promise((resolve, reject) => {
+      await new Promise((resolve: any, reject: any): any => {
```

Comment:
can make this return void, should also add a type that is PromiseRes and PromiseRej since we do this a *lot*